### PR TITLE
Add Remove plugin from server

### PR DESCRIPTION
**Pull request by merhuman**

### CODE REVIEW

File: src/atlassian/jira.py [hunk 1]
```diff
@@ -99,3 +99,24 @@ def get_plugin_pending_task_info(self, task_id: str) -> dict:
             raise ValueError(msg)
 
         return res_json
+
+    def remove_plugin(self, plugin_key: str):
+        """
+        Delete plugin
```
Comment (jackblk) [inline src/atlassian/jira.py]: Please use [Google style docstring](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings)

**Review by jackblk — CHANGES_REQUESTED** [review]
LGTM! Good first PR :).

Just that we need to use Google docstring style to conform with the repo.

File: src/atlassian/jira.py (hunk 1 above)
Comment (merhuman) [reply src/atlassian/jira.py]: Fixed

File: src/atlassian/jira.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import json
 from pathlib import Path
+from xmlrpc.client import boolean
```
Comment (jackblk) [inline src/atlassian/jira.py]: this is a mistake I think 😆

File: src/remove_plugin.py [hunk 3]
```diff
@@ -1,7 +1,98 @@
...
+    vars = {
+        "url": url if url else Config.ATLAS_URL,
+        "username": username if username else Config.ATLAS_USERNAME,
+        "password": password if password else Config.ATLAS_PASSWORD,
+    }
+    for key, val in vars.items():
+        if val == "":
+            raise ValueError(f"'{key}' is not set")
+
+    if len(notify) == 0 and Config.NOTIFY_TITLE != "":
```
Comment (jackblk) [inline src/remove_plugin.py:42]: it should be `Config.NOTIFY_URL`, I fixed it in https://github.com/itdevsamurai/atlassian-plugin-remote-installation-docker/commit/6f8c458898411a0f6f08302583143a507e1b4eee

**Pull request merged**